docs/use-cases/portfolio-management/README.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_navigation:
+  title: OpenProject Use-Cases: Portfolio Management and Custom Reporting
+  priority: 999
+description: Step-by-step instruction about portfolio management and custom reporting
+robots: index, follow
+keywords: help, documentation, use-cases, portfolio management
+---
+# Use Cases:
+
+# Portfolio Management and Custom Reporting Options
+
+Step 1: To view all projects, first select the *project chooser drop-down*, the chose *View all projects*
+
+![Chose project](chose-project.jpg)
+
+ 
+
+Step 2: You can sort and filter this view using custom fields, such as *Initiative* or *Status.* If you have not added useful custom fields, please see: https://www.openproject.org/docs/system-admin-guide/custom-fields/
```

Comment:
Please use the markdown link format [title](relative path).
`[Custom fields](../../system-admin-guide/custom-fields/)`